core/shared/src/main/scala/monocle/Lens.scala

```diff
@@ -224,6 +224,12 @@ object PLens extends LensInstances {
     )(t => _.bimap(_ => t, _ => t))
 
   /**
+   * create a [[PLens]] from a context (indexed store) coalgebra.
+   */
+  def contextCoalg[S, T, A, B](f: S => IndexedStore[A, B, T]): PLens[S, T, A, B] =
```

Comment:
The thing is that after introducing `IndexedStore`, `f` is no longer a *coalgebra* (`S => F[S]`). It becomes something different (`S => F[T]`). There aren't many references about this thing, but at least in the lens context, it seems that the right name for `f` should be [*coalgebroid*](http://r6.ca/blog/20120623T104901Z.html). On the other hand, I know that `IndexedStore` is also known as "context", but I don't know to what extent. Honestly, I don't like `contextCoalg` much, so please, feel free to rename it to whatever you like.